Tensile/KernelWriterAssembly.py

```diff
@@ -7116,10 +7149,25 @@ def localReadDo(self, kernel, bufferIdx, iui, epsi, tP):
 
             valuIdx += blockWidth
 
-            if kIdx % 2 == 1 and kIdx > 0:
+            if kIdx % 2 == 1:
               # pack 2 half-dword into one
               destVgpr = vgpr("Valu%s_X%u_I%u+%u" % (tc, bufferIdx, iui, rIdx))
               imod.addInst("s_waitcnt lgkmcnt(0)", "")
+              if kernel["MatrixInstruction"] and \
```

Comment:
"MatrixInstruction" was already checked above

Reply:
Will clean that up in the next PR. Thanks!